packages/metrics/src/Metrics.ts

```diff
@@ -480,6 +485,13 @@ class Metrics extends Utility implements MetricsInterface {
     return Object.keys(this.storedMetrics).length > 0;
   }
 
+  /**
+   * Whether metrics are disabled.
+   */
+  public isDisabled(): boolean {
+    return this.disabled;
+  }
```

Comment:
This method could be omitted, but it followed the style from Logger which also exposes the `getLogEvent` method.
Also, this allows to easily test the effect of different environment variables.

Reply:
Agree with making it a method, although I would prefer if we can make it `protected` rather than `public`.

I think it's fair to make it accessible to those who extend the Metrics class, but I wouldn't necessarily open it up for the public API.